test/message/error_aggregateTwoErrors.out

```diff
@@ -1,34 +1,15 @@
 *error_aggregateTwoErrors.js:*
 throw aggregateTwoErrors(err, originalError);
 ^
-AggregateError: original
-    at Object.<anonymous> (*test*message*error_aggregateTwoErrors.js:*:*)
-    at Module._compile (node:internal/modules/cjs/loader:*:*)
-    at Module._extensions..js (node:internal/modules/cjs/loader:*:*)
-    at Module.load (node:internal/modules/cjs/loader:*:*)
-    at Module._load (node:internal/modules/cjs/loader:*:*)
-    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:*:*)
-    at node:internal/main/run_main_module:*:* {
+[AggregateError: original] {
```

Comment:
Not sure why this change is needed?

Reply:
I don’t understand why this output changed either.